docs/sources/enterprise/access-control/permissions.md

```diff
@@ -68,6 +76,7 @@ The following list contains fine-grained access control actions.
 | `server.stats:read`              | n/a                                                                                         | Read server stats                                                                                                                                          |
```

Comment:
```suggestion
| `server.stats:read`              | n/a                                                                                         | Read server stats                                                                                                                                          |
```
This is supposed to be "instance statistics", right? Because this is also for Cloud, no?